### DISCUSSION TITLE
有什么渠道可以为蓝灯专业版续费?

### DISCUSSION THREAD

**Discussion by weijingwei** [getlantern/lantern-forum-cn#857] [General]
之前购买蓝灯专业版激活码的页面无法访问了. 有什么渠道可以购买到蓝灯专业版激活码吗?或者支付宝,微信支付的方式也可以. 联系邮箱: weijingwei@sina.com

**Comment by XINYUAN1986** [comment]
看论坛里发现最近很多账号无法登录专业版，问题尚未修复，暂时不建议购买，静候处理结果。

**Reply by MaxMKMax** [reply]
@XINYUAN1986 这个是系统问题. 我们一般会帮用户解决登入专业版的问题. 请您放心!

**Reply by XINYUAN1986** [reply]
**我的WIN7系统专业版无法登录，到现在也没解决。**

**Reply by Lewis0000221** [reply]
我的专业版买了两个星期，手机一直用不了，连接后谷歌页面都打开不了，申请退款也没后续

**Comment by MaxMKMax** [comment]
@weijingwei 我帮您找支付系统团队弄个支付链接给您. 不好意思让您久等了.

**Reply by haopc** [reply]
你好，我也无法完成购买，选择支付宝支付后，二维码一闪而过，怎么解决？

**Reply by LuLu120325** [reply]
你好，我也想购买一年的专业版，可以发一下支付链接嘛？

**Reply by LuLu120325** [reply]
我的邮箱 2647611833@qq.com

**Reply by MaxMKMax** [reply]
@LuLu120325 我帮您找支付系统团队弄个支付链接给您. 不好意思让您久等了.

**Reply by MaxMKMax** [reply]
@haopc 您好. 请您提供我账号邮箱. 我让支付团队给您弄个支付链接. 谢谢.
请问您是安卓系统手机吗? 还是什么系统? 电脑还是手机?

**Reply by MaxMKMax** [reply]
@haopc 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by LuLu120325** [reply]
你好，我已经购买成功，请问是怎么升级呢，邮箱还没收到激活码，谢谢

**Reply by LuLu120325** [reply]
已经成功登录并升级为专业版，非常感谢！

**Comment by MaxMKMax** [comment]
@weijingwei 
您好. 这是给您账号购买专业版的链接.
https://buy.stripe.com/14k7v577O0Qq4s8eV3

**Comment by LuLu120325** [comment]
你好，我已经购买成功，这是支付截图￼

**Comment by weijingwei** [comment]
@Mike777Khoo 您好, 我已购买成功. 但是现在无法登录专业版账号. weijingwei@sina.com

**Comment by greySkiesMarsh** [comment]
@weijingwei Hello, 

For some reason, I'm not able to locate an account associated with that email address in our system. Please redeem this code, and let me know which plan you purchased and I can adjust your account to reflect the correct expiration date.

GFFJP-XXX9D-F2DDK-FWCJ9-RQ4VJ

**Comment by weijingwei** [comment]
意思是找不到 ***@***.*** 这个账号了吗? 而且我注意到, 我在其他设备上的 lantern 都失去了专业版授权.
我在9月12号购买过一次 2年+3月 版的. 在这之前 (购买前) 的过期时间是今年年底, 大概是2024年12月. 总的过期时间应该是在2027年3月

----- Original Message -----
From: Marsh ***@***.***>
To: getlantern/lantern-forum-cn ***@***.***>
Cc: 魏经纬 ***@***.***>,  Mention ***@***.***>
Subject: Re:_[getlantern/lantern-forum-cn]_有什么渠道可以为蓝灯专业版续费?_(Discussion_#857)
Date: 2024-09-24 00:38


@weijingwei Hello,
For some reason, I'm not able to locate an account associated with that email address in our system. Please redeem this code, and let me know which plan you purchased and I can adjust your account to reflect the correct expiration date.
GFFJP-XXX9D-F2DDK-FWCJ9-RQ4VJ

**Comment by weijingwei** [comment]
另 附上 9月12号的支付记录


----- Original Message -----
From: Marsh ***@***.***>
To: getlantern/lantern-forum-cn ***@***.***>
Cc: 魏经纬 ***@***.***>,  Mention ***@***.***>
Subject: Re:_[getlantern/lantern-forum-cn]_有什么渠道可以为蓝灯专业版续费?_(Discussion_#857)
Date: 2024-09-24 00:38


@weijingwei Hello,
For some reason, I'm not able to locate an account associated with that email address in our system. Please redeem this code, and let me know which plan you purchased and I can adjust your account to reflect the correct expiration date.
GFFJP-XXX9D-F2DDK-FWCJ9-RQ4VJ

**Comment by LuLu120325** [comment]
你好，我的账号升级后现在又变成了免费版，并且不能通过邮件授权，麻烦解决一下，谢谢！

**Reply by weijingwei** [reply]
我也是类似的问题. 回复我的邮件是系统里找我到我的账号了. 所以收不到授权邮件

**Comment by LuLu120325** [comment]
希望可以帮我反馈解决这个问题，还是很想继续用lantern的




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月26日(星期四) 下午4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 有什么渠道可以为蓝灯专业版续费? (Discussion #857)






我也是类似的问题. 回复我的邮件是系统里找我到我的账号了. 所以收不到授权邮件

**Comment by Alby-777** [comment]
你好，想续费 续不了呀，怎么办呀？

**Comment by Alby-777** [comment]
![xf](https://github.com/user-attachments/assets/fe48ec3a-89ec-412a-bf1d-bf9013e10ce0)
这个提示
